license-scan/src/main.rs

```diff
@@ -396,7 +396,7 @@ fn write_attribution(
         if let Some(stated_license) = stated_license {
             stated_license.to_string()
         } else {
-            licenses.sort();
+            licenses.sort_unstable();
```

Comment:
This is fine, but I'm mostly just curious what prompted this change.

Reply:
Nothing terrible. Clippy just suggested the switch for performance reasons.